src/main/resources/application.yaml

```diff
@@ -0,0 +1,35 @@
+dfspids: "DFSPID"
+
+transaction-id-length: -1
+
+mpesa:
```

Comment:
change mpesa to airtel connector, also I think this can be removed for now

Reply:
Won't we need mpesa for gorrila tenant? Airtel connector will decide which service to use(airtel/mpesa) based on the tenant? 

I have removed the code block for now.

Reply:
the above specified if notifcations was enabled for the flow or not (refer the mpesa roster bpmn), we don't have notifications in our bpmn so remove this. Also the tenant based payment schema config(mpesa or airtel) should be a part of channel connector and not here.

Reply:
Got it.